R/input-time-series.R

```diff
@@ -503,12 +503,7 @@ prepare_input_time_series_anc <- function(anc, shape) {
 
   anc_plot_data_long <- anc_long |>
     dplyr::mutate(!!!mutate_exprs) |>
-    dplyr::select(area_id, area_name, area_level, area_level_label, parent_area_id,
-                  area_sort_order, age_group,  time_period, year, quarter,
-                  calendar_quarter, anc_clients, anc_tested, anc_tested_pos,
-                  anc_prevalence, anc_known_pos, anc_known_neg,
-                  anc_art_coverage, births_facility, births_clients_ratio,
-                  area_hierarchy) |>
+    dplyr::select(-sex) |>
```

Comment:
Is there any risk that `anc_long` has other columns dangling around, or are the columns of that table rigidly specified by an earlier step?

Reply:
So the indicators are specified by mutate_exprs. So not selecting specific columns here means if we add any more then they should come straight through. Are you worried about some new non-indicator column though?

Reply:
Double checked, no that shouldn't happen. We read the anc file in and specify the columns we want to read. So that should filter out any unwanted stuff. This should just mean all the columns we create during the aggregation get returned automatically